tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+        "target": "ES5",
+        "module": "commonjs",
+        "resolveJsonModule": true,
+        "esModuleInterop": true,
+        "sourceMap": true,
+        "outDir": "build/dist",
```

Comment:
A common typescript idiom is to build to the lib directory. We can rename lib to src, outDir to lib, and git ignore the lib folder after the rename.

Reply:
I believe but don't fully recall that this is a longer running transpiled package idiom like with coffeescript/etc.

Reply:
You suggested moving the build ts from build/dist to lib and the source to src. If I do that I end up getting the built code located at lib/src which I think looks really weird and confusing. I cannot just remove the src dir in the built package because we also need to pull in package.json (for the version #) into the built package. Should I still try and follow this idiom?

```
~/airtable/airtable.js% ls lib/**/*(.)                  
lib/package.json                    lib/src/deprecate.js.map                        lib/src/package_version.js.map
lib/src/abort-controller.d.ts       lib/src/exponential_backoff_with_jitter.js      lib/src/promise.js
lib/src/abort-controller.js         lib/src/exponential_backoff_with_jitter.js.map  lib/src/promise.js.map
lib/src/abort-controller.js.map     lib/src/fetch.js                                lib/src/query.js
lib/src/airtable.d.ts               lib/src/fetch.js.map                            lib/src/query.js.map
lib/src/airtable_error.d.ts         lib/src/has.js                                  lib/src/record.js
lib/src/airtable_error.js           lib/src/has.js.map                              lib/src/record.js.map
lib/src/airtable_error.js.map       lib/src/http_headers.js                         lib/src/run_action.js
lib/src/airtable.js                 lib/src/http_headers.js.map                     lib/src/run_action.js.map
lib/src/airtable.js.map             lib/src/internal_config.json                    lib/src/table.js
lib/src/base.js                     lib/src/object_to_query_param_string.js         lib/src/table.js.map
lib/src/base.js.map                 lib/src/object_to_query_param_string.js.map     lib/src/typecheck.js
lib/src/callback_to_promise.js      lib/src/package_version_browser.js              lib/src/typecheck.js.map
lib/src/callback_to_promise.js.map  lib/src/package_version_browser.js.map
lib/src/deprecate.js                lib/src/package_version.js
```

Reply:
I fixed this issue by removing `package.json` from the `tsconfig`.